.gitignore

```diff
@@ -1 +1,4 @@
 /target
+Cargo.lock
+.gitignore
```

Comment:
Probably don't need to gitignore gitignore
```suggestion

```

Reply:
And don't need to gitignore Cargo.lock:
https://stackoverflow.com/questions/43667176/what-files-in-a-cargo-project-should-be-in-my-gitignore